js/actions/appActions.js

```diff
@@ -268,13 +268,16 @@ const appActions = {
    * @param {Boolean} activateIfOpen if the tab is already open with the same properties,
    * switch to it instead of creating a new one
    * @param {Boolean} isRestore when true, won't try to activate the new tab, even if the user preference indicates to
+   * @param {Boolean} focusWindow
+   * @param {Boolean} allowFile - When true, allows file:// URLs to be opened
    */
-  createTabRequested: function (createProperties, activateIfOpen = false, isRestore = false, focusWindow = false) {
+  createTabRequested: function (createProperties, activateIfOpen = false, isRestore = false, focusWindow = false, allowFile = false) {
```

Comment:
style comment (not blocking by any means 😛 I know it was already like this) - when functions start to have this many params, it might be better to accept a map with named params